accelerator/apis.c

```diff
@@ -1,43 +1,73 @@
 #include "apis.h"
 
+static status_t char_buffer_to_str(char_buffer_t* char_buffer_input, char** str_output) {
```

Comment:
this file should contain api only. If you have trouble reusing function, you shall re-think the structure and find better place to implement.

Reply:
@wusyong I keep it in the plain way